authenticate/handlers.go

```diff
@@ -534,11 +528,11 @@ func (a *Authenticate) Dashboard(w http.ResponseWriter, r *http.Request) error {
 }
 
 func (a *Authenticate) saveSessionToDataBroker(ctx context.Context, sessionState *sessions.State, accessToken *oauth2.Token) error {
-	if a.sessionClient == nil || a.userClient == nil {
+	if a.dataBrokerClient == nil {
 		return nil
 	}
 
-	sessionExpiry, _ := ptypes.TimestampProto(time.Now().Add(time.Hour))
+	sessionExpiry, _ := ptypes.TimestampProto(time.Now().Add(time.Hour * 24 * 365))
```

Comment:
Should we set this to be equal to the total lifetime of the session instead of a year? We could use a new option for this value,  -- or the existing -- cookie expiry value (which I think defaults to a week) and has the same intent.

Reply:
edit: we actually set cookies to expire after 14 hours.

https://github.com/pomerium/pomerium/blob/3039407597d9b510faf7559f0c8d8d78ce5e7e4a/config/options.go#L263